src/parser/mod.rs

```diff
@@ -8452,6 +8458,42 @@ impl<'a> Parser<'a> {
         }
     }
 
+    /// Parse a literal unicode normalization clause
+    pub fn parse_unicode_is_normalized(
+        &mut self,
+    ) -> Result<(Option<NormalizationForm>, bool), ParserError> {
+        let neg = self.parse_keyword(Keyword::NOT);
+        if self.parse_keyword(Keyword::NORMALIZED) {
+            return Ok((None, neg));
+        }
+        let index = self.index;
+        let next_token = self.next_token();
+        let normalized_form = if let Token::Word(Word {
+            value: ref s,
+            quote_style: None,
+            keyword: Keyword::NoKeyword,
+        }) = next_token.token
+        {
+            match s.to_uppercase().as_str() {
+                "NFC" => Some(NormalizationForm::NFC),
+                "NFD" => Some(NormalizationForm::NFD),
+                "NFKC" => Some(NormalizationForm::NFKC),
+                "NFKD" => Some(NormalizationForm::NFKD),
+                _ => {
+                    self.index = index;
+                    return self.expected("unicode normalization", next_token);
+                }
+            }
+        } else {
+            None
+        };
+        if self.parse_keyword(Keyword::NORMALIZED) {
+            return Ok((normalized_form, neg));
+        }
+        self.index = index;
```

Comment:
Thinking it might be simpler to declare the strings as keywords, also instead of manually tracking the index in the function would it be possible to use `maybe_parse`? roughly thinking something like:
```rust
let normalized_form = self.maybe_parse(|parser| {
	match parser.parse_one_of_keywords(&[
		Keyword::NFC, ...
	]) {
		Some(Keyword::NFC) = Some(NormalizationForm::NFC)
		...
	}
})?;
if self.parse_keyword(Keyword::NORMALIZED) {
	return Ok((normalized_form, neg));
}
```

Reply:
Nice hadn't previously spotted `maybe_parse`; I wasn't sure if making the normalization forms keywords would be seen as "noisy", but the end result definitely seems cleaner. Have updated the code accordingly ✌️